lib/autoupdate/agent/installer.go

```diff
@@ -223,7 +223,7 @@ func (li *LocalInstaller) Install(ctx context.Context, rev Revision, baseURL str
 		return trace.Wrap(err, "failed to extract teleport")
 	}
 	// Write the checksum last. This marks the version directory as valid.
-	err = renameio.WriteFile(sumPath, []byte(hex.EncodeToString(newSum)), configFileMode)
+	err = os.WriteFile(sumPath, []byte(hex.EncodeToString(newSum)), configFileMode)
```

Comment:
This checksum never needed to use renameio, as a corrupt checksum results in the same behavior as a missing checksum.